webapp/app/containers/Widget/components/Chart/Table/index.tsx

```diff
@@ -40,7 +40,8 @@ import styles from '../Chart.less'
 
 import {
   findChildConfig, traverseConfig,
-  computeCellWidth, getDataColumnWidth, getMergedCellSpan, getTableCellValueRange } from './util'
+  computeCellWidth, getDataColumnWidth, getMergedCellSpan, getTableCellValueRange
```

Comment:
Please reset these lines like space puzzles to its original.

Reply:
Use the formatting tool built into vscode  `shift+alt+f`